tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/promo_block.html

```diff
@@ -1,17 +1,16 @@
 {% load wagtailimages_tags %}
 
 <div class="grid__promo promo-block">
-    <div class="promo-block__content">
-        <h3>{{ value.title }}</h3>
-        <p>{{ value.description }}</p>
-        <a href="{{ value.get_button_link }}" class="button">{{ value.button_text }}</a>
+    <section class="promo-block__content">
```

Comment:
We've not used `section` tags for other blocks, so please use `div` here for consistency.

Reply:
Changed in fae1889f39d83c142f8832a6001ee69ebdfcb834